src/sentry/interfaces/exception.py

```diff
@@ -64,10 +65,11 @@ def to_python(cls, data, slim_frames=True):
         type = data.get('type')
         value = data.get('value')
         if isinstance(value, six.string_types):
-            if type is None and ':' in value.split(' ', 1)[0]:
-                type, value = value.split(':', 1)
-                # in case of TypeError: foo (no space)
-                value = value.strip()
+            if type is None:
+                m = re.match('^(\w+):(.*)$', value)
```

Comment:
we should precompile regexes as we hit the regex cache already which is really dumb

Reply:
done